client/src/pages/CommunityPage/CommunityPageGroup.js

```diff
@@ -113,22 +81,90 @@ class CommunityPageGroup extends Component {
     console.log(val);
   };
 
+  commentClick = val => {
+    console.log(val);
+  };
+
+  likeClick = val => {
+    console.log(val);
+  };
+
+  handleSelect = event => {
+    let userSubmission = event.target.textContent.toLowerCase();
+
+    this.state.groupList.map(group => {
+      if (userSubmission === group.title.toLowerCase()) {
+        this.setState({ groupId: group.id }, this.getGroup);
+      }
+    });
+  };
+
   render() {
+    console.log(getAllGroups);
+    let groupList = this.state.groupList;
+
+    // TO DO: Media Section, Upcoming Events Section
```

Comment:
If you remove the space here (`TODO` instead of `TO DO`), most editors will recognize this and we can see these in a list throughout the codebase

Reply:
Done

Reply:
The following code relates to Miles' addition, so passing off edits to him. @MilesHamilton